### ISSUE TITLE
Owner Dimlet

### ISSUE THREAD

**Issue by PapaWulfy** [McJty/RFTools#2132]
Versions:
* Minecraft: 1.7.10
* Forge: 1.7.10.1614
* RFTools: one from infinity evolved. sorry not sure

Just wondering how would I disable the finding of owner dimlet through the  dimension researcher?

**Comment by McJty** [comment]
The owner dimlet can never be found through the dimension researcher

**Comment by PapaWulfy** [comment]
Odd, one of my players just reported finding one in a researcher and he wanted to alert me as we only sell them in shop.

**Comment by McJty** [comment]
If they can be found in the researcher then that means there was some misconfiguration. i.e. the owner dimlet is probably configured to be findable then. But by default it isn't

**Comment by PapaWulfy** [comment]
Sorry to ask, but would you know where this is configured? For 1.7 we have two configurations. I can't seem to find the owner dimlet spawn options

**Comment by McJty** [comment]
Should be dimlets.cfg I think. Has been a while since I looked at 1.7

**Comment by PapaWulfy** [comment]
So would 0 mean disabled? Or how does the ranking system work

**Comment by McJty** [comment]
Show me what it says

**Comment by PapaWulfy** [comment]
Whole Config: https://pastebin.com/fmfbarGM

TLDR: 
I have these options for "Owner" inside dimlets.cfg
*     I:rfmaintain.Special.Owner=0
*     I:rfcreate.Special.Owner=0
*     I:rarity.Special.Owner=0
*     I:ticks.Special.Owner=0

**Comment by McJty** [comment]
Hmm, 1.7.10 is really too long ago for me so I don't remember how it worked. But did you do manual configuration of your dimlets? Can you maybe send me all your rftools related configs?

**Comment by PapaWulfy** [comment]
https://pastebin.com/Mexs86Ue I unfortunately only have 1 more. This is main.cfg

**Comment by McJty** [comment]
And my other question?

**Comment by PapaWulfy** [comment]
No sorry, I have not touched the dimlets whatsoever

**Comment by McJty** [comment]
Weird. No idea I'm afraid. That dimlet should not appear from the researcher. But 1.7.10 is really too long ago for me and it is possible that there is a bug

**Comment by MichaelHillcox** [comment]
Gunna close this as it's looking like it's just a bug in a very old version, we no longer support any versions before 1.12. Please feel feee to add more info you feel it's relevant
